udata/core/dataset/rdf.py

```diff
@@ -603,7 +614,17 @@ def dataset_from_rdf(graph: Graph, dataset=None, node=None):
 
     identifier = rdf_value(d, DCT.identifier)
     uri = d.identifier.toPython() if isinstance(d.identifier, URIRef) else None
-    remote_url = remote_url_from_rdf(d)
+
+    # compute remote_url, either with specified prefix or directly from RDF
+    catalog_record_identifier = catalog_record_identifier_from_rdf(graph)
```

Comment:
Should we check whether the identifier is already an URI? In which case it's probably better to skip it?
cc @alhyss

Reply:
Nevermind, I mixed up some notes about representing `fileIdentifier` as a `gmx:Anchor`. Even if we were to handle that case (which we haven't seen in practice), the contents of the node would likely still be an id, not an URI.